src/npcmove.cpp

```diff
@@ -3279,8 +3279,8 @@ void npc::do_reload( item &it )
                                        it.ammo_capacity() - it.ammo_remaining() ) );
     int reload_time = item_reload_cost( it, *usable_ammo, qty );
     // @todo: Consider printing this info to player too
-    const std::string ammo_name = usable_ammo->tname();
```

Comment:
This is a bug, usable_ammo is no longer valid for deference after the next line.